index.bs

```diff
@@ -18,6 +18,19 @@ Markup Shorthands: dfn yes
 Markup Shorthands: idl yes
 Markup Shorthands: css no
 Logo: https://webmachinelearning.github.io/webmachinelearning-logo.png
+Status Text: <p class="note" role="note">
+  Further implementation experience and user feedback is being gathered for the
+  {{MLCommandEncoder}} interface that proposes to enable more efficient WebGPU
+  interoperability and the related proposal to
```

Comment:
I'm not #322 is directly related to WebGPU interop, so I think that needs rephrasing.

Also, I know we've adopted the phrase "WebGPU interoperability" but I think "WebGPU integration" is a better description of what we're looking for feedback on.

Reply:
Thanks, s/interoperability/integration might be better and consistent with the charter text too.

I want to hear @wchao1115's proposal how to phrase this before updating this PR.

Reply:
Agreed with @dontcallmedom. I don't think #322 is related to WebGPU interop, or more specifically the `MLCommandEncoder`. This text is a bit misleading.

Reply:
Would you prefer to exclude `MLCommandEncoder` from this note entirely? Feel free to propose a better wording. We want this text to be clear.

Reply:
The note on `MLCommandEncoder` should remain as we do need more data from implementation experience. We also anticipated that the WebGPU folks will have substantive feedback on it. 

But it's a different topic from #322 and I think it's best to keep them separate to avoid confusing people.

re: #322, in my view it's a bit strange to ask for an implementation feedback on something that is proposed to be removed from the spec, rather than something to be added. The proposed removal of the internal GPU device would, by definition, require no implementation for it. 

As a thought exercise, I think it seems far more intuitive to remove this feature from the spec, add it as an experimental feature, then ask the developers for their feedback (of implementing it).

Reply:
The next iteration is up for your review. Please take a look and suggest further refinements as needed. It helps if we can be concise while not oversimplifying things so improvements in that regard are especially welcome.

Reply:
@wchao1115 PR updated with a more concise version of this note, PTAL!